codegen-core/src/main/kotlin/io/vrap/rmf/codegen/di/GeneratorModule.kt

```diff
@@ -71,7 +71,7 @@ class GeneratorModule constructor(
             LOGGER.warn("Could not find proper package name configuration. Using default $defaultPackage")
             return defaultPackage
         }
-        return generatorConfig.basePackageName ?: URI(api.baseUri.expand()).host.split(".").reversed().joinToString("/")
+        return generatorConfig.basePackageName ?: defaultPackage
```

Comment:
this was done by intention to have as the generated SDK and other generated files should live in the namespace of the API spec.

Reply:
This actually started throwing some exceptions in case of some URLs, also the URLs we use now are too long, i know we can override it via the base package, but i would just let it to the default package which is reasonable and short. and if someone wants to modify it, he has to override it.

Reply:
But then maybe we should change the default package name ;)

Reply:
any suggestions 👀

Reply:
Or we change it to only use the first 2 entries of the reversed hostname. Which would mean: com.commercetools

Reply:
ok, this might actually work, but we should check if the baseURL is provided to be safe 😢